src/frontend/overlay/src/components/atoms/ContestCells.js

```diff
@@ -24,6 +24,21 @@ export const formatScore = (score, digits = 2) => {
     return score?.toFixed((score - Math.floor(score)) > 0 ? digits : 0);
 };
 
+export const formatPenalty = (contestInfo, penalty) => {
+    if (penalty === undefined) {
+        return "";
+    }
+    let mode = contestInfo.penaltyRoundingMode;
+    if (mode === "sum_in_seconds" || mode === "last") {
+        return Math.floor(penalty / 60) + ":" + (penalty % 60);
```

Comment:
I need some help here. First of all, this is not correct as it prints 192:6 instead of 192:06. 
But it also too wide for this cell. Can you please fix it?